tests/integration/bigquery.spec.ts

```diff
@@ -67,7 +67,25 @@ suite("@dataform/integration/bigquery", ({ after }) => {
     ]);
 
     // Run the project.
-    let executionGraph = await dfapi.build(compiledGraph, {}, credentials);
+
+    const actionsToRun = [
+      ...compiledGraph.tables
+        .filter(
+          table => table.name !== "dataform-integration-tests.df_integration_test.depends_on_example_view"
```

Comment:
why not just run it? for that matter, we already have actions that depend on `example_view`, why not get rid of it entirely and use one of those?